internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -11,57 +11,113 @@ PublicNetworkLoadBalancer:
             !Sub '${AppName}-${EnvName}-PublicSubnets'
     Type: network
 
+{{- if eq (len .NLB.Listener) 1 }}
+{{- range $listener := .NLB.Listener }}
```

Comment:
oooh I wonder if instead of a big if-else statement we could write it this way:
```
{{- range $i, $listener := .NLB.Listener }}
{{if eq $i 0}}
NLBListener:
{{- else }}
NLBListener{{$i + 1}}:
{{- end }}
```

This way if there is a bug fix we don't need to remember to update the properties of the same resource in two different places

Reply:
Yeah you are right, I thought you are suggesting me to keep existing logic as is. But I agree we can just have one block doing both the conditions.